templates/blog/article.html

```diff
@@ -0,0 +1,177 @@
+{% extends "base_index.html" %}
+
+{% block title %}{{ article.title.rendered|safe }}{% endblock %}
+{% block meta_description %}{{ article.excerpt.raw }}{% endblock %}
+{% block meta_image %}{{ article.image.source_url }}{% endblock %}
+
+{# Change canonical URL for "snapcraft.io" articles (2996 is the ID of the "snapcraft.io" tag) #}
+
+{% block body_class %}blog-article{% endblock %}
+{% block canonical_url %}{% if 2996 in article.tags %}https://snapcraft.io/blog/{{ article.slug }}{% elif 4059 in article.tags %}https://juju.is/blog/{{ article.slug }}{% else %}{{ super() }}{% endif %}{% endblock canonical_url %}
+
+{% block extra_metatags %}
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@id": "https://ubuntu.com/#article",
+    "@type": "Article",
+    "name": "{{ article.title.renderered|safe }}",
+    "headline": "{{ article.excerpt.raw }}",
+    "author": {
+       "@type": "Person",
+       "name": "{{ article.author.name }}"
+    },
+    "datePublished": "{{ article.date_gmt }}",
+    {% if article.image and article.image.source_url %}
+      "image": "{{ article.image.source_url }}",
+    {% endif %}
+    "url": "{{ request.url }}",
+    "publisher": {
+      "@type": "Organization",
+      "name": "Ubuntu"
+    }
+  }
+  </script>
+{% endblock %}
+
+{% block content %}
+<article>
+  <header class="p-strip--suru">
+    <div class="row">
+      <div class="col-8">
+        <h1>{{ article.title.rendered|safe }}</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-8">
+        <div class="p-media-object">
+          {% if article.author %}
+            <img src="{% if article.author.id == 217 %}https://assets.ubuntu.com/v1/60d9b81e-picto-canonical.svg{% else %}{{ article.author.avatar_urls['96'] }}{% endif %}" class="p-media-object__image is-round" alt="">
+          {% endif %}
+
+          <div class="p-media-object__details">
+            {% if article.author %}
+              <h3 class="p-media-object__title">
+                <a href="/blog/author/{{ article.author.slug }}" title="More about {{ article.author.name }}">{{ article.author.name }}</a>
```

Comment:
This link should have the [inverted styling](https://vanillaframework.io/docs/patterns/links#inverted).

Reply:
Not applicable anymore as we are using the light background header